### PR TITLE
fix(styles): Checked radio buttons is wrong on dark background

### PR DESCRIPTION
**Pull request by imagoiq**
## What

This PR is a bit confusing as it addresses two problems.
1. Checked radio button were creating a visual shift in inline layout. It was fixed with https://github.com/swisspost/design-system/pull/2086 by using a border instead of a margin. But it created the second following problem.
2. Checked radio button on dark background blends the circle indicator with the background. #2170

Other additions:
- Added inline layout to snapshot

### Before

image

image

### After

image

image

### CODE REVIEW

File: packages/styles/src/components/form-check.scss [hunk 1]
```diff
@@ -78,7 +78,7 @@
       border-radius: 50%;
 
       &:checked::after {
-        margin: spacing.$size-micro;
+        border: spacing.$size-micro solid color.$white;
```
Comment (gfellerph) [inline packages/styles/src/components/form-check.scss:81]: If a default background color scheme was defined (https://github.com/swisspost/design-system/issues/2123), we could use 

```suggestion
        border: spacing.$size-micro solid rgb(var(--post-bg-rgb));
```
and have automatic dark/light background compatibility.

**Review by gfellerph — CHANGES_REQUESTED** [review]
Do we want to wait for the changes in #2123?

**Comment by imagoiq** [comment]
> Do we want to wait for the changes in #2123?

I'm not sure anymore as it's a bug. I think that it should be quickly fixed.

File: .changeset/gorgeous-gorillas-confess.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-styles': patch
+---
+
+Fixed offset of radio button when checked and in inline layout.
```
Comment (imagoiq) [inline .changeset/gorgeous-gorillas-confess.md:5]: As the second problem is caused by the first and that the first was not announced in a previous changelog, I'll stick to this sentence about the first problem.

File: packages/styles/src/components/form-check.scss [hunk 3]
```diff
@@ -83,9 +83,10 @@
       border-radius: 50%;
 
       &:checked::after {
-        border: spacing.$size-micro solid color.$white;
```
Comment (imagoiq) [inline packages/styles/src/components/form-check.scss:86]: The border addition was erroneously introduced in https://github.com/swisspost/design-system/pull/2086

File: packages/components/.eslintrc.js [hunk 4]
```diff
@@ -11,6 +11,7 @@ module.exports = {
   ],
   overrides: [],
   parser: '@typescript-eslint/parser',
+  ignorePatterns: ['.eslintrc.js'],
```
Comment (imagoiq) [inline packages/components/.eslintrc.js:14]: Fix unit tests from: https://github.com/swisspost/design-system/pull/2238/files#diff-4ade8993db0ab98cef66268e7a71355b158eefa4db4a3e03f623638800612980R14

**Review by gfellerph — APPROVED** [review]

**Pull request merged**